docker-compose/docker-compose.platform.yml

```diff
@@ -1,10 +1,10 @@
-version: "3.5"
+version: "3.6"
 services:
   configmanager:
     build:
       context: ..
       labels:
-        iotea.platform.version: "0.3.0"
+        iotea.platform.version: "0.5.0"
```

Comment:
Please add and align versions in the CHANGELOG.md file (root directory) and list the changes there.

I'm struggling with such an high version jump since we have no release for far I see it more like a 0.4.x. What do you think?

Reply:
I've discussed the version increase with Lars:
[5/10, 10:10 AM] Ivanova Maria
    Do you mind if I raise it to 0.4.2 with the protocol gateway branch
​[5/10, 10:15 AM] Kiefer Lars-Erich
    Actually it should be 0.5.0, because the gateway thing is actually a new feature (actually a breaking one, but since we have no official major release yet, it's ok to have it as minor version).

Regarding the CHANGELOG, I guess we need to have js-sdk and py-sdk to 0.5.0 but no version change to vscode-sdk, what's your opinion?

Reply:
Good proposal 👍  (js and py-sdk to 0.5.0, no vscode-sdk change)